src/natcap/invest/ndr/ndr_core.pyx

```diff
@@ -68,19 +73,21 @@ def ndr_eff_calculation(
     # cell sizes must be square, so no reason to test at this point.
     cdef float cell_size = abs(stream_info['pixel_size'][0])
 
-    cdef _ManagedRaster stream_raster = _ManagedRaster(stream_path, 1, False)
-    cdef _ManagedRaster crit_len_raster = _ManagedRaster(
-        crit_len_path, 1, False)
+    cdef ManagedRaster stream_raster = ManagedRaster(
+        stream_path.encode('utf-8'), 1, False)
```

Comment:
I imagine once we have the python C interface for logging figured out that we might be able to do this string encoding as well!